ssl/storage-proxy/s3.docker.mender.io.crt

```diff
@@ -0,0 +1,18 @@
+-----BEGIN CERTIFICATE-----
+MIIC6jCCAdICCQDjVN7VrrdKlDANBgkqhkiG9w0BAQsFADA3MRcwFQYDVQQKDA5N
+ZW5kZXJTb2Z0d2FyZTEcMBoGA1UEAwwTczMuZG9ja2VyLm1lbmRlci5pbzAeFw0x
+NzAxMDIxMTAxMjJaFw0yNjEyMzExMTAxMjJaMDcxFzAVBgNVBAoMDk1lbmRlclNv
+ZnR3YXJlMRwwGgYDVQQDDBNzMy5kb2NrZXIubWVuZGVyLmlvMIIBIjANBgkqhkiG
+9w0BAQEFAAOCAQ8AMIIBCgKCAQEAyeZEpHvAE1YqyNKt29x5MYKUlHPgRuh8v0KD
+HpfmDSo71l2r8yspwd4Taf95oQf9NYxa+qT5x0thwKAo6XztVXtWEb51fkKd2uCm
+sGJeJN14ta0QU0TF42Gxdip1djLNOots3Fy+bBa6CylmGP0ihPeJufz4TKDbsnrp
+0cWVpKsKsjVV9eaoze3nhZzp5O57AVzEhFTMzMopvyj1iLs9RhADO6ARIjAZmAcA
+3tdsC9hEChgMczUKykhVqfD7Efx2DJEbj1WqOv+wxPcF/urWGz6ltlUBM2lRbLHc
+0uIzJ+Q1LKyIHKZ+mj4+2kZxmwn27VY8VlPaI2HNPVBsho+uowIDAQABMA0GCSqG
+SIb3DQEBCwUAA4IBAQAWAiLBAdiTa4vB0J/OZDLg4ioCkACivIO93DVq5T8u0K9/
+u1GTfz0eaOiwspsPofjrlYq8StD6FvXzy+MMinp3JFBEH7FKksm+P+oXTpMcRGPH
+OLLdUYqwjFHViQqO30UnQvUjW4y7wjxgqyjuZTb3t4m5h0ovXLd2wVufkLH1pavk
+VPxZJc2KeZ0shHCeuxgPTbz+36Ein/H1dI4cKZ9ia7pcxIGIIgxmEmus09QiUrA3
+BhMrGSWvnXP/E62GeHd1/IfbnyECQ+vqb7T0U9HFCeBSnDKuP/lerbejZ4zPhpPr
+SizKxVb2vBhVGvfahnVMkB+CaCSwC7g2+4lY16f3
+-----END CERTIFICATE-----
```

Comment:
Is this the key that should be added to the client as per https://tracker.mender.io/browse/MEN-920?